falkordb-cluster/cluster-entrypoint.sh

```
@@ -416,6 +426,9 @@ run_node() {
   sed -i "s/\$FALKORDB_RESULT_SET_SIZE/$FALKORDB_RESULT_SET_SIZE/g" $NODE_CONF_FILE
   sed -i "s/\$FALKORDB_QUERY_MEM_CAPACITY/$FALKORDB_QUERY_MEM_CAPACITY/g" $NODE_CONF_FILE
   sed -i "s/\$FALKORDB_VKEY_MAX_ENTITY_COUNT/$FALKORDB_VKEY_MAX_ENTITY_COUNT/g" $NODE_CONF_FILE
+  sed -i "s/\$FALKORDB_PASSWORD/#$FALKORDB_PASSWORD/g" $ACL_CONF_FILE
+  sed -i "s/\$FALKORDB_USER/$FALKORDB_USER/g" $ACL_CONF_FILE
+  sed -i "s/\$ADMIN_PASSWORD/#$ADMIN_PASSWORD_HASH/g" $ACL_CONF_FILE
   echo "dir $DATA_DIR/$i" >>$NODE_CONF_FILE
```

Shellcheck (0.10.0)

[warning] i is referenced but not assigned.

(SC2154)